swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -95,162 +95,120 @@ Follow the instructions given in this section to develop the service.
+
+        ```ballerina
+        service /healthcare on new http:Listener(8290) {
+            resource function post categories/[string category]/reserve(ReservationRequest payload) 
+                    returns http:Created|http:NotFound|http:InternalServerError {
+                
+            }
+        }
+        ```
+
+5. Define a [configurable variable](https://ballerina.io/learn/by-example/#configurability) for the URL of the backend service.
 
-   - The `ReservationRequest` record uses [record type inclusion](https://ballerina.io/learn/by-example/type-inclusion-for-records/) in the type of the `patient` field to include all the fields from the `Patient` record along with the `cardNo` field.
+    > **Note:** Enable two factor authentication on your Google account, generate an app password, and use the app password in place of your email password. The app password can be generated with [this link](https://myaccount.google.com/apppasswords?pli=1&rapt=AEjHL4Mf5XDD4rE79YJpP5E2NoNwhvXMET_TWyBcQRn-HMzt0PI8BmptpMGRiBVIamW-0ECgVZtXxMRA19bL4Wfnq_hmjBEMqA).
 
-   - The initial record definitions can be generated using the "Paste JSON as record" VSCode command with the relevant JSON payloads and the records can then be modified as necessary.
+    ![Define a configurable variable](/learn/images/integration-tutorials/sending-a-message-to-a-service/define_a_configurable_variable.gif)
 
-6. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource function that accepts user requests, makes calls to the backend services to retrieve relevant details, and sends emails to the client with appointment details.
+    Similarly define configurable variables for the payment backend and host, username and password of the SMTP client.
+
+    The generated code will be as follows.
 
     ```ballerina
-    service /healthcare on new http:Listener(port) {
-        resource function post categories/[string category]/reserve(ReservationRequest payload) 
-                returns http:Created|http:NotFound|http:InternalServerError {
-            
-        }
-    }
+    configurable string hospitalServicesBackend = "http://localhost:9090";
+    configurable string paymentBackend = "http://localhost:9090/healthcare/payments";
+    configurable string host = "smtp.gmail.com";
+    configurable string username = ?;
+    configurable string password = ?;
     ```
 
-   - Use `/healthcare` as the service path (or the context) for the service attached to the listener that is listening on port `port`.
-     
-   - The HTTP resource allows the `POST` operation on resource path `/categories/{category}/reserve`, where `category` (corresponding to the specialization) is a path parameter.
+6. Define an [`http:Client`](https://ballerina.io/learn/by-example/#http-client) object to send requests to the backend service.
+   
+   ![Define the clients](/learn/images/integration-tutorials/sending-emails-from-a-service/define_clients.gif)
```

Comment:
We may have to crop in the middle of the video. But let's wait for other reviews and then do.